app/models/comment.rb

```diff
@@ -1,4 +1,12 @@
 class Comment < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  before_save :update_comments_counter
```

Comment:
- [ ] You can use the after_save to call the #update_comments_counter method every time a comment has been added 👍🏿